addons/web/static/src/legacy/js/views/basic/basic_model.js

```diff
@@ -2914,31 +2914,12 @@ var BasicModel = AbstractModel.extend({
             return Promise.resolve();
         }
 
-        var self = this;
         return this._rpc({
                 model: field.relation,
                 method: 'search_read',
-                fields: ["id"].concat(fieldsToRead || []),
+                fields: ["id", "display_name"].concat(fieldsToRead || []),
```

Comment:
aren't you at risk to ask twice the display_name if it was in the fieldsToRead ? I don't know if that can be an issue (and technically already existed for the id)

Reply:
I just tested and even if there are duplicates in `fields list` the orm doesn't care and behave the same as if there is no duplicate.
I admit it's not elegant, maybe a `[...new Set([...["id", "display_name"] ,...fieldsToRead])]` instead would be better. What do you think ?

Reply:
I'd like @aab-odoo opinion on this.

Reply:
but that's not really important, let's keep it like this for now as it keeps the code simpler if there is no bad side effect

Reply:
I don't think it's a big deal (also, it's legacy code). We could also remove "id", as it is always returned anyway.